VideoWeb/VideoWeb.UnitTests/Hub/SendHeartbeatTests.cs

```diff
@@ -20,6 +20,8 @@ public async Task Should_send_heartbeat_to_vhofficers_group()
             var participantUsername = "individual@test.com";
             var conference = InitConference(participantUsername);
             var participant = conference.Participants.First(x => x.Username == participantUsername);
+            var judge = conference.Participants.First(x => x.Role == Role.Judge);
```

Comment:
Please ensure you have a test to send to judge and VHO if participant is not a judge also
May be better to update the test name so that it is reflective of the new behaviour

Reply:
the exisiting test id for participant, have updated for judge to receive heratbeat from participants